04_numeronym/kasunfdo/main.go

```diff
@@ -0,0 +1,37 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func numeronyms(s ...string) []string {
+	out := make([]string, len(s))
+	for i, str := range s {
+		out[i] += getNumeronym(str)
```

Comment:
The `+=` here seems unnecessary here. The slice is initialised to `len(s)` with each element being the zero value for a string - i.e. the empty string. Using `+=` on a known empty string doesn't really make much sense.